public/stylesheets/style.css

```diff
@@ -0,0 +1,727 @@
+* {
+  box-sizing: border-box; }
+
+html
+body {
+  font-family: 'Dosis';
+  font-weight: 600;
+  font-size: 20px;
+  margin: 0; }
+
+svg {
```

Comment:
why we have added global rule like this which can add to all SVGs?

Reply:
According to project requirement all svg images have same color so added globally